sqlglot/parser.py

```diff
@@ -4083,6 +4084,17 @@ def _parse_concat(self) -> t.Optional[exp.Expression]:
             exp.Concat if self.STRICT_STRING_CONCAT else exp.SafeConcat, expressions=args
         )
 
+    def _parse_concat_ws(self) -> t.Optional[exp.Expression]:
+        delim, *args = self._parse_csv(self._parse_conjunction)
```

Comment:
in general the parser is pretty lenient, so in ignore mode, this shouldn't raise an error because there are no args, for example, concat_ws() should not raise a python error here

Reply:
this is pretty much copy and pasted it from concat maybe let's make parse_concat take an argument

Reply:
The function is pretty simple so it feels a bit overkill to creating branchinig behavior in `_parse_concat`. I did though put the section where we COALESCE the values into a dedicated method that they both call since that logic was directly repeated. 

In terms of the first comment, this doesn't work for me: `parse_one("CONCAT()")`. Is that expected?

Reply:
you need to set the error_level to ignore